src/rulesets/Base/fastmath_able.jl

```diff
@@ -133,7 +133,7 @@ let
 
         @scalar_rule x + y (One(), One())
         @scalar_rule x - y (One(), -1)
-        @scalar_rule x / y (inv(y), -((x / y) / y))
+        @scalar_rule x / y (one(Ω) / y, -(Ω / y))
```

Comment:
Because `d/dx (x) = one(x)`, shouldn't this be `one(x)`?  It probably doesn't matter in general but could in some cases. e.g. if `x` is real but `y` is complex, your version will have more operations than this one.
```suggestion
        @scalar_rule x / y (one(x) / y, -(Ω / y))
```

Reply:
I did that because the following thought crossed my mind: that a closure referencing only Ω  could be more memory efficient in some situations than one referencing both x and Ω. I have no idea if in practice this is actually true, could be?

Reply:
Hm maybe? I don't know enough about the performance of closures.

Reply:
I think performance wise those should not pose too much of a difference

Reply:
Yeah, I still prefer for `one(x)`, but I won't hold this PR up over it.